### DISCUSSION TITLE
Website development help

### DISCUSSION THREAD

**Discussion by GivieJnr** [community/community#142796] [Programming Help]
### Body

I  am a new b and i  need web developers to help me on my project

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LuckyMan564** [comment]
Hi.
I am a full stack website developer.
So if you have any problems, don't hesitate to invite me at any time.
Good luck!

**Reply by GivieJnr** [reply]
I will definitely,  I don't know how to connect my project on github, but let me figure out first,  I live in France

**Reply by LuckyMan564** [reply]
Hi.
Nice to meet you.
You can contact me via zhiqiangzhang547@gmail.com.
Have a nice day.

**Comment by ebndev** [comment]
Hi @GivieJnr, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about your project / code snippets or a link to your repo. Giving a few more details might help someone give you a nudge in the right direction. 😄

**Reply by GivieJnr** [reply]
Thanks guys I need at least 5 people who are willing to help me,  I am building a website for kindergarten home schooling for a village school in Zimbabwe

**Comment by aziz-codes** [comment]
Let's build something really cool together  <3

**Reply by GivieJnr** [reply]
How are you,  thanks guys I am going to link my project on github

**Comment by Ksgf452** [comment]
您好，虽然我帮不到您什么，但还是祝你成功，加油！

**Reply by joegeorge022** [reply]
nice🤝

**Comment by joegeorge022** [comment]
Let's do it!
I'll try my best to help.

**Comment by SimoAcharouaou777** [comment]
Hi , i'm a Full Stack Web Developer and i will be happy to help

**Comment by LuckyMan564** [comment]
Hi.
Thanks for your email.
Let's make your dreams come true together!
Please DM me asap.

**Comment by Ohhprojunction** [comment]
In today’s digital-first world, your website is often the first point of contact with potential customers. It’s essential that your site not only looks great but also functions seamlessly to drive engagement and conversion. At  Techtinklers, we specialize in crafting custom websites that are optimized for performance, user experience, and search engines. Our [website development services](https://www.techtinklers.com/) ensure that your site is built with the latest technologies, is mobile-responsive, secure, and scalable to grow with your business needs.

**Comment by DylanKatz786** [comment]
If you need website development help, make sure to focus on both design and functionality. I used a [free audit SEO tool](https://serpwizz.com/) to improve my site’s performance and SEO during a recent development project!
